### PR TITLE
Temporarily disable static site upload in staging and prod

### PR DESCRIPTION
**Pull request by cwinters-usds**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -394,9 +394,11 @@ jobs:
           az functionapp config access-restriction remove -g ${{ env.RESOURCE_GROUP }} -n ${{ env.PREFIX }}-functionapp --slot candidate --rule-name GitHubActionIPV4 > /dev/null 2>&1
 
       - name: Upload Static Site
-        run: |
-          az storage blob delete-batch --account-name ${{ env.PREFIX }}public -s '$web'
-          az storage blob upload-batch --account-name ${{ env.PREFIX }}public -d '$web' -s frontend-react/build
+        working-directory: frontend-react/build
+        run: ls -R frontend-react/build
```
Comment (cwinters-usds) [inline .github/workflows/release.yml:398]: ugh fixing this...

Comment (cwinters-usds) [reply .github/workflows/release.yml:397]: fixed

Comment (ronaldheft-usds) [reply .github/workflows/release.yml:397]: Debug info?

**Review by ronaldheft-usds — APPROVED** [review]
Looks good assuming `ls` statement is for debugging

**Comment by cwinters-usds** [comment]
@ronaldheft-usds yep just to see where those files are

**Pull request merged**